internal/httpclient/api_metadata.go

```diff
@@ -14,7 +14,7 @@ package client
 import (
 	"bytes"
 	"context"
-	"io/ioutil"
+	"io"
```

Comment:
Hm, the auto-generated files will always use `io/ioutil`. I propose to add some automation to `make sdk` that fixes this after the SDK is regenerated.

Reply:
It would make more sense to update the OpenAPI Generator template

Reply:
Yup, you are right, I forgot about the generated code.. 

I'll have a look at them and try to fix it there. Are they defined by us or upstream in the generator?

I would like to avoid some automatic tooling for this and rather fix it in the generation logic.

Reply:
Done.